inputs.tf

```diff
@@ -9,10 +9,10 @@ variable "name_suffix" {
   default     = ""
 }
 
-variable "input_tags" {
-  description = "Map of tags to apply to resources"
-  type        = map(string)
-  default     = {}
+variable "enable_cloudwatch_logs_for_cloudtrail" {
+  default     = false
```

Comment:
If we change the module major version this is fine, otherwise it should be `true`